src/esn.calendar.libs/app/components/event/form/event-form.controller.js

```diff
@@ -325,6 +325,9 @@ function CalEventFormController(
     if ($scope.event.rrule && !$scope.event.rrule.equals($scope.editedEvent.rrule)) {
       $scope.editedEvent.deleteAllException();
     }
+    if ($scope.event.alarm && !$scope.editedEvent.alarm) {
+      $scope.editedEvent.alarm = $scope.event.alarm;
+    }
```

Comment:
```suggestion
   $scope.editedEvent.alarm = $scope.editedEvent.alarm  || $scope.event.alarm;
```

Reply:
Writing tests would be better to make PR becomes reasonable and maintainable.

Reply:
about the suggestion it does not work on my case , the  existing tests always fail because it will set alarm to undefined when the user did'nt insert a value for it